src/lib/stores.ts

```diff
@@ -12,10 +13,13 @@ const noopStorage = {
 // whether we are running on server vs client
 const isBrowser = typeof document !== 'undefined';
 
+// the modes that are supported, used for validation & type derivation
+export const modes = ['dark', 'light', 'system'] as const;
+
 /**
  * The key used to store the mode in localStorage.
  */
-export const localStorageKey = 'mode';
+export const localStorageKey = 'mode-watcher-mode';
```

Comment:
`'mode-watcher-mode'` does not match the key used in `mode-watcher.svelte`

I have no strong preference for either name, so I assume you wanted to go with this name instead and that `mode-watcher.svelte` should be updated

Reply:
Yeah, we just need to import the var into that component and use it. The reason I'm being more descriptive here is we want to prevent any conflicts with any other local storage things users may have on their apps. `mode` could be used for a wide range of things, so better to be specific and distance from common keys.